Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.h

```diff
@@ -47,7 +47,7 @@ namespace AZ
             AZStd::vector<AZStd::pair<RPI::Cullable::LodOverride, AZStd::string>> GetLodOverrideValues();
 
             Data::Asset<RPI::ModelAsset> m_modelAsset = { AZ::Data::AssetLoadBehavior::QueueLoad };
-            RHI::DrawItemSortKey m_sortKey = 0;
+            AZ::s64 m_sortKey = 0;
```

Comment:
Changing `RHI::DrawItemSortKey` to `AZ::s64` would be better but changing it here reduces the scope of the change and addresses the serialization issue. Outside of serialization, using `int64_t` or `uint64_t` should not cause problems.

Reply:
I'd say make an GHI for changing the `RHI::DrawItemSortKey` to the appropriate underlying type (`AZ::s64`), but also add a commented out "`RHI::DrawItemSortKey`" here so that search-replace can find this one.  If this change is approved for stabilization then it will reach development eventually too.

Reply:
Made the change and created an issue: https://github.com/o3de/o3de/issues/9314

Reply:
Cant we just do this? I am unclear how this will require further changes
using DrawItemSortKey = int64_t;
to
using DrawItemSortKey = AZ::s64;

Reply:
Yeah. But since that increased the scope of the change and this is targeting the stabilization branch, I wasn't sure if everyone would be on board. If graphics is ok with the change, I can just change `DrawItemSortKey` instead.

Reply:
You solution will work and seems ok to me,
I would prefer:
using DrawItemSortKey = AZ::s64;
But as this is in Atom, they should have the final ok.

Reply:
Unless someone else objects I would be ok with doing
using DrawItemSortKey = AZ::s64;
Just make sure to test ASV full suite using dx12 and Vk. I can help you with steps on how to do that.

Reply:
Updated to just fix DrawItemSortKey.

Reply:
Thanks. Just make sure to the fulltestsuite to ensure nothing is broken from the change.